### PR TITLE
complete and correct code comment

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/discovery/discovery_client.go [hunk 1]
```diff
@@ -440,7 +440,7 @@ func NewDiscoveryClientForConfigOrDie(c *restclient.Config) *DiscoveryClient {
 
 }
 
-// New creates a new DiscoveryClient for the given RESTClient.
+// NewDiscoveryClient return  a new DiscoveryClient for the given RESTClient.
```
Comment (piosz) [inline staging/src/k8s.io/client-go/discovery/discovery_client.go]: returns

**Review by piosz — COMMENTED** [review]
Looks good, one type

**Comment by piosz** [comment]
@lavalamp could you please approve?

**Comment by piosz** [comment]
ping @lavalamp

**Comment by piosz** [comment]
@wojtek-t could you please approve?

**Comment by zouyee** [comment]
@wojtek-t  PTAL

**Pull request merged**